.github/cocogitto-changelog-template

```diff
@@ -0,0 +1,56 @@
+{# Using a literal newline to set the newline variable -#}
+{% set newline = "
+" -%}
+
+{% set commit_base_url = repository_url ~ "/commit/" -%}
+{% set compare_base_url = repository_url ~ "/compare/" -%}
+{% set pr_base_url = repository_url ~ "/pull/" -%}
+
+{% if version.tag -%}
+    ## {{ version.tag | trim_start_matches(pat="v") }} ({{ date | date(format="%Y-%m-%d") }})
+{% else -%}
+    {% set from = from.id -%}
+    {% set to = version.id -%}
+
+    {% set from_shorthand = from.id | truncate(length=7, end="") -%}
+    {% set to_shorthand = version.id | truncate(length=7, end="") -%}
+
+    ## Unreleased ([`{{ from_shorthand ~ ".." ~ to_shorthand }}`]({{ compare_base_url ~ from_shorthand ~ ".." ~ to_shorthand }}))
+{% endif -%}
+
+{% for type, typed_commits in commits | sort(attribute="type") | group_by(attribute="type") %}
+{# The `striptags` removes the HTML comments added while grouping -#}
+### {{ type | striptags | trim | upper_first }}
+{% for scope, scoped_commits in typed_commits | group_by(attribute="scope") %}
+- {{ "**" ~ scope ~ ":" ~ "**" -}}
+    {% for commit in scoped_commits | sort(attribute="date") -%}
+    {% set shorthand = commit.id | truncate(length=7, end="") -%}
+    {% set commit_link = commit_base_url ~ commit.id -%}
+    {# Replace PR numbers in commit message with PR link -#}
+    {% set pr_number = commit.summary | split(pat="(#") | last | trim_end_matches(pat=")") -%}
```

Comment:
This may be problematic in case of other `)` in the summary
```
trim_end_matches[🔗](https://tera.netlify.app/docs/#trim-end-matches)

Removes trailing characters that match the given pattern if the variable is a string.

Example: {{ value | trim_end_matches(pat="//") }}

If value is "//a/b/c//", the output will be "//a/b/c".
```

Reply:
The commit summary (first line of commit message) has the PR title and the PR number in the following format:

`{pr_title} (#pr_number)`

The other lines in the commit message are available in the `commit.body` and `commit.footer` fields.